QualityUnit/postaffiliatepro-theme
# [694] LA elements and new default shadow

**Changes proposed in this Pull Request**
Boxes styles from LA web
Default box shadow was changed

**Testing instructions**
Check styles of new boxes elements from LA web (https://www.liveagent.com/collab-alternative/)
Check shadows on PAP theme pages

**Checklist**

Close QualityUnit/web-issues#694
